code/modules/vehicles/janicart.dm

```diff
@@ -34,30 +34,30 @@
 		return
 	..()
 
-/obj/vehicle/train/cargo/engine/pussywagon/turn_on()
+/obj/vehicle/train/cargo/engine/janicart/turn_on()
 	if(!key)
 		return
 	else
 		..()
 		update_stats()
 
-		verbs += /obj/vehicle/train/cargo/engine/pussywagon/verb/toggle_mop
-		verbs += /obj/vehicle/train/cargo/engine/pussywagon/verb/toggle_hoover
+		verbs += /obj/vehicle/train/cargo/engine/janicart/verb/toggle_mop
+		verbs += /obj/vehicle/train/cargo/engine/janicart/verb/toggle_hoover
 
-/obj/vehicle/train/cargo/engine/pussywagon/turn_off()
+/obj/vehicle/train/cargo/engine/janicart/turn_off()
 	..()
 
 	if(tow)
-		if(istype(tow,/obj/vehicle/train/cargo/trolley/pussywagon))
-			var/obj/vehicle/train/cargo/trolley/pussywagon/PW = tow
+		if(istype(tow,/obj/vehicle/train/cargo/trolley/janicart))
+			var/obj/vehicle/train/cargo/trolley/janicart/PW = tow
```

Comment:
Those initials translates as Pussy Wagon. Imagine if a coder comes across those filthy initials when trying to improve janicarts and get mad so he decides to abandon codebase, leaving players without features. But that's only a tip of iceberg. Imagine if triggered so much that he will decide to harm himself, did you EVER think about it? Pls do a proper job and remove ALL of those FILTHY, MISOGYNISTIC references.